test/integration/certificates/issuing_controller_test.go

```diff
@@ -80,6 +80,12 @@ func TestIssuingController(t *testing.T) {
 		secretName               = "test-crt-tls"
 	)
 
+	// Create Namespace
+	_, err := kubeClient.CoreV1().Namespaces().Create(context.TODO(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}, metav1.CreateOptions{})
```

Comment:
Where were these namespaces coming before? Why are they now being created here now?

Reply:
The test environment used to create these on making the objects that behaviour was removed